src/main/java/com/google/cloud/genomics/dataflow/utils/CallSetNamesOptions.java

```diff
@@ -65,11 +65,15 @@
      * @throws IOException
      */
     public static StreamVariantsRequest getRequestPrototype(final CallSetNamesOptions options) throws IOException {
-      return StreamVariantsRequest.newBuilder()
-          .setProjectId(options.getProject())
+      StreamVariantsRequest.Builder request = StreamVariantsRequest.newBuilder()
           .setVariantSetId(options.getVariantSetId())
-          .addAllCallSetIds(getCallSetIds(options))
-          .build();
+          .addAllCallSetIds(getCallSetIds(options));
```

Comment:
Hi Nicole,

Doesn't this request require a `build()` or will it be built later?

Thanks,
Paul